src/components/home/About.jsx

```diff
@@ -1,5 +1,50 @@
+import React from "react";
+import Title from "../Title";
+import Subtitle from "../Subtitle";
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="p-56 mx-96 flex flex-col items-center">
```

Comment:
do you need such a big padding?
don't use mx, use a fraction width to control the horizontal space like w-9/12